package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "library-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+
+  }
+  ,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/javierbotero/libraryProject.git"
+  }
+  ,
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+
+  }
+  ,
+  "homepage": "https://github.com/javierbotero/libraryProject#readme",
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0"
+    ,
+    "eslint-plugin-import": "^2.20.2"
+    ,
+    "stylelint": "^13.3.3"
+    ,
+    "stylelint-config-standard": "^20.0.0"
+    ,
+    "stylelint-csstree-validator": "^1.8.0"
+    ,
+    "stylelint-scss": "^3.17.2"
+  }
+  ,
```

Comment:
- [ ] When I run `npm install`, it runs into an error (screenshot below) because of the improper syntax linting highlighted in the `package.json` file above. It also generates wrong JSON syntaxes in the `package-lock.json` file ( you can take a look into this file to see what I am saying) which could not parse as a JSON. 

You should note that special attention is always taken when writing JSON data, e.g - like you cannot have a commented code in it and the syntaxes must be properly used and placed in the right way. In the case of the highlighted code, the commas should be in the same line with the key-pair values. 

In order for me to run `npm install`, I had to delete `package-lock.json`, corrected the above-highlighted code, and ran `npm install` again to generate another `package-lock.json` file. 
Kindly take your time to look into this and get the issue fixed.

![Screenshot from 2020-10-01 22-55-08](https://user-images.githubusercontent.com/50773261/94867489-3a23be00-0439-11eb-993b-78167d4d3be3.png)